quarkus/admin/src/main/resources/application.properties

```diff
@@ -23,7 +23,7 @@
 # Cf. https://quarkus.io/guides/all-config
 
 quarkus.application.name=Apache Polaris Admin Tool (incubating)
-quarkus.banner.path=/org/apache/polaris/admintool/banner.txt
```

Comment:
Please also remove the banner.txt file :)

Reply:
Removed